api/HTMLMediaElement.json

```diff
@@ -966,10 +966,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "20"
+              "version_added": false
```

Comment:
Where did the original "20" come from? This is the only thing being set to false in this PR. I suspect it's correct, but warrants a closer look.

Reply:
The original data had come from the initial wiki migration, it looks like.

Reply:
OK, thanks!